platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1267,6 +1312,28 @@ public void onResponse(Call call, Response response) throws IOException {
         return true;
     }
 
+    // Called from JNI on worker or render-thread.
+    void sceneReadyCallback(final boolean success, final SceneUpdateError error) {
+        if (!success && sceneUpdateErrorListener != null) {
```

Comment:
Add comment/notes to mark this depricated and to remove when we remove `SceneUpdateErrorListener` in a major releases.